src/types.ts

```diff
@@ -0,0 +1,9 @@
+import { PeripheralOptions } from './views/nodes/peripheralnode';
```

Comment:
More candidates for a common api file maybe?

Reply:
I moved them to `api.ts` file.